### PR TITLE
Added new option to prevent autofocus on input search

### PR DESCRIPTION
**Pull request by JoseVte**
Due to issues on mobile devices, and using the code from #1467 I added a new option on the Vue component

### CODE REVIEW

File: src/multiselectMixin.js [hunk 1]
```diff
@@ -657,8 +666,8 @@ export default {
       /* istanbul ignore else  */
       if (this.searchable) {
         if (!this.preserveSearch) this.search = ''
-        this.$nextTick(() => this.$refs.search && this.$refs.search.focus())
-      } else {
+        this.$nextTick(() => this.$refs.search && (!this.preventAutofocus && this.$refs.search.focus()))
```
Comment (akki-jat) [inline src/multiselectMixin.js:669]: Please check

https://github.com/shentao/vue-multiselect/pull/1544#discussion_r1000963718

File: build/vue-loader.conf.js [hunk 2]
```diff
@@ -11,7 +11,7 @@ module.exports = {
   }),
   postcss: [
     require('autoprefixer')({
-      browsers: ['last 2 versions']
+      overrideBrowserslist: ['last 2 versions']
```
Comment (akki-jat) [inline build/vue-loader.conf.js:14]: Why this change was added?

**Comment by akki-jat** [comment]
Please remove all build files as they will be generated while creating package. No need to add them.

File: build/vue-loader.conf.js (hunk 2 above)
Comment (JoseVte) [reply build/vue-loader.conf.js:14]: because when I'm compiling on dev, npm suggests do this change

Comment (akki-jat) [reply build/vue-loader.conf.js:14]: Why we are compiling on dev? We should just create PR for required changes only and compiling will be done when we publish it.

Comment (akki-jat) [reply build/vue-loader.conf.js:14]: Docs files are fine as those are needed to update docs but we can skip dist ones

**Comment by akki-jat** [comment]
Opened new PR to fix merge conflicts - https://github.com/shentao/vue-multiselect/pull/1579
